content/en/blog/2021-10-01-introducing-whisper.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Introducing Whisper"
+slug: "introducing-whisper"
+date: 2021-10-01T16:32:47-04:00
+draft: true
+image_webp: images/blog/whispering_wheat.webp
+image: images/blog/whispering_wheat.jpg
+author: Edwin Schmierer
+description: "Shhh..there's a better way to share secrets. Introducing Whisper, our secret-sharing utility."
+---
+Let's imagine a few scenarios. You have a new engineer joining your team and you need to provide passwords for access to critical systems. Or you need to send a sensitive file to a client. Or you need to share encryption keys with a team member. What do you do?
+
+You might text a password or key (hopefully using [Signal](https://signal.org/en/) since it uses end-to-end encryption), but that won't work for lengthy keys, files, or large docs. You might write it on a piece of paper or place it on thumb drive, but both are limited by proximity, which frankly doesn't work too well when remote work has become the norm. Or you might email it but that exposes several security risks. What if the recipient forwards it or is hacked or never deletes the message so it remains indefinitely in their inbox?
```

Comment:
a thumb drive

Reply:
How secure if your email, really? If you're using a popular free email service, the ToS might give the provider permission to read all your emails!